core/include/sched.h

```diff
@@ -179,10 +179,10 @@ NORETURN void sched_task_exit(void);
  *  Scheduler statistics
  */
 typedef struct {
-    unsigned int laststart;         /**< Time stamp of the last time this thread was
```

Comment:
uint64_t would be more semantically correct since _xtimer_now64 returns an uint64_t